docs/source/assets/css/vendor/scooter.scss

```diff
@@ -294,6 +294,9 @@ a,
   border: 1px solid currentColor;
   border-radius: 3px;
   cursor: pointer; }
+  .c-btn:not(button, [type=submit]) {
+    /*! You must ensure the element matches one of these selectors: button, [type=submit] */
+    outline: 2px solid red !important; }
   .c-btn[disabled], .c-btn.c-btn--disabled {
```

Comment:
Good question. Now that we’re enforcing/asserting the element is an actual `button`, we can probably get rid of the disabled modifier.